### DISCUSSION TITLE
Duplicate Projects in Contember Cloud with Studio

### DISCUSSION THREAD

**Discussion by osseonews** [contember/admin#594] [Support] [answered]
Been investigating Cloud with Studio for a project and the problem we ran into is that every single time we use the Studio to edit the project ("Show in Studio"), and make some changes to the schema there, and then rebuild, an entirely new duplicate project is created in the Cloud. We care getting tons of duplicate projects on every change. If we delete the duplicate project, then the original project ceases to work also. So not really sure how you can use Studio on Cloud after the original project is created to edit it, add to the schema etc. Are we doing something wrong? Thanks.

**Comment by jonasnobile** [comment]
Hi @osseonews, unlocking a project and editing it after deployment is an experimental feature in Studio. We are working on ways to allow editing of existing projects and deploying them to cloud, but we are at the beginning. So right now, if you want to edit a deployed project in Studio, you will end up with a duplicate project in Cloud. 

Your duplicated project shouldn't stop when you delete other projects. If you send me a project slug, organisation name or e-mail used in our Cloud, we can investigate what is happening to your projects.


The best way to work with Studio and our cloud right now is:

1. start your project with AI in Studio
2. make the first obvious changes to the app structure
3. deploy and test
4. connect to Github repo
5. clone Github repo locally and continue development locally

Sometimes it makes sense to start over and try different words and phrases. You may get a better result.

**Comment by sitole** [comment, marked as answer]
Studio now supports project re-deployments into same project in Contember Cloud so you can deploy, then adjust some things in studio and deploy new version that contains admin re-deployment and optionally if needed schema migrations.
